src/main/java/NEWTsPresenter.java

```diff
@@ -0,0 +1,148 @@
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import json.Spell;
+import json.SpellList;
+
+import javax.swing.*;
+import java.util.Random;
+
+public class NEWTsPresenter
+{
+    private NEWTsPracticeExam view;
+    private SpellGenerator model;
+    private Disposable disposable;
+
+    boolean spellSelected = false;
+    String name = null;
+    String incantation = null;
+    String effect = null;
+    String category = null;
+
+    int totalAsked = 0;
+    int totalCorrect = 0;
+
+    public NEWTsPresenter(NEWTsPracticeExam view, SpellGenerator model)
+    {
+        this.view = view;
+        this.model = model;
+    }
+
+    public void resetFlashCard(String category)
+    {
+        if (category.equals("--"))
+        {
+            view.setEffect("Effect Goes Here");
+            view.resetIncantation();
+            view.setResult("");
```

Comment:
combine these methods into one.

Reply:
Even though I use these elsewhere separated? 

Or as a sort of additional resetView?

Reply:
Have an individual method in the view call all 3 if necessary.